SmartDeviceLink/SDLProtocol.h

```diff
@@ -80,6 +93,13 @@ extern NSString *const SDLProtocolSecurityErrorDomain;
  */
 - (void)endServiceWithType:(SDLServiceType)serviceType;
 
+/**
+ *  Sends a Register Secondary Transport control frame to Core
+ *
+ *  @param payload The data to send in the message
+ */
+- (void)registerSecondaryTransport:(nullable NSData *)payload;
```

Comment:
This should be `registerSecondaryTransportWithPayload:`, but this is only ever called with `nil`, so the parameter is unnecessary.

Reply:
OK, removed with commit b4455699e2f589b9e46db24e96e86015575f6442.
The `payload` parameter was added in case we want to extend the control frame in future. Now that `SDLProtocol` class is not public, we can freely update the code whenever we want to extend.